tests/robustness/linearizability_test.go

```diff
@@ -41,7 +41,7 @@ func TestRobustness(t *testing.T) {
 		scenarios = append(scenarios, testScenario{
 			name:      "ClusterOfSize1/" + traffic.Name,
 			failpoint: RandomFailpoint,
-			traffic:   &traffic,
+			traffic:   traffic,
```

Comment:
Alternative way that does not depend on loop variable.

```golang
traffics := []traffic.Config{traffic.LowTraffic, traffic.HighTraffic, traffic.KubernetesTraffic}
for i, _ := range traffics {
   		scenarios = append(scenarios, testScenario{
			name:      "ClusterOfSize1/" + traffic.Name,
			failpoint: RandomFailpoint,
			traffic:   &traffics[i],
                }
}
```

Reply:
Yea, having interface in config is not great. In ideal place you want configuration to be serializable and independent of implementation.